src/any_agent/tools/wrappers.py

```diff
@@ -76,21 +71,7 @@ async def wrap_mcp_server(
     based on the specified agent_framework
     """
     # Select the appropriate manager based on agent_framework
```

Comment:
💅 nitpick: (not part of this PR, but just spotted it now) I would drop this comment